lib/src/controls/navigation/tree_view.dart

```diff
@@ -478,6 +483,10 @@ class TreeView extends StatefulWidget {
   /// item to be the same size as the first item in the tree view.
   final bool usePrototypeItem;
 
+  /// Whether or not to have narrow spacing between the contents of each item.
+  /// The default value is `false`.
+  final bool narrowSpacing;
+
   @override
   _TreeViewState createState() => _TreeViewState();
```

Comment:
Just for convenience

```suggestion
  State<TreeView> createState() => _TreeViewState();
```